src/pint/observatory/__init__.py

```diff
@@ -415,7 +431,7 @@ def compare_t2_observatories_dat(t2dir=None):
     """Read a tempo2 observatories.dat file and compare with PINT
 
     Produces a report including lines that can be added to PINT's
-    observatories.py to add any observatories unknown to PINT.
+    observatories.json to add any observatories unknown to PINT.
```

Comment:
Should this be tweaked to output JSON instead of python?

Reply:
Not really. This was a tool I used to compare with TEMPO and TEMPO2 so I could resolve any differences between PINT and the other two, so I just pasted the right things in. You could check that entries were correctly parsed by your reader, but you'd have to fiddle the registry first to produce an apparent discrepancy.